h/static/scripts/karma.config.js

```diff
@@ -57,10 +59,16 @@ module.exports = function(config) {
       extensions: ['.coffee'],
       noParse: [require.resolve('./vendor/katex')],
       configure: function (bundle) {
-        bundle
-          .transform('coffeeify')
-          .plugin('proxyquire-universal');
+        bundle.plugin('proxyquire-universal');
       },
+
+      transform: [
+        'coffeeify',
+        istanbul({
+          ignore: ['**/node_modules/**', '**/katex.js', '**/*.html', '**/*.svg'],
```

Comment:
According to [the docs](https://github.com/devongovett/browserify-istanbul#usage) these are additions to the default ignores, which includes `node_modules`, so I don't think we need to specify those here.

Is the decision to ignore just katex.js from `scripts/vendor` rather than just ignoring the whole of that directory intentional - since we're not going to be writing unit tests for Annotator but instead eventually removing it as a dependency.